app/controllers/api/v1/admin/domain_allows_controller.rb

```diff
@@ -19,12 +19,8 @@ class Api::V1::Admin::DomainAllowsController < Api::BaseController
   def create
     authorize :domain_allow, :create?
 
-    @domain_allow = DomainAllow.find_by(resource_params)
-
-    if @domain_allow.nil?
-      @domain_allow = DomainAllow.create!(resource_params)
-      log_action :create, @domain_allow
-    end
```

Comment:
This was to make the API call idempotent. I don't think there's harm either way.